deployment/terraform/modules/osv/variables.tf

```diff
@@ -12,3 +12,13 @@ variable "vulnerabilities_export_bucket" {
   type        = string
   description = "Name of bucket to export vulnerabilities to."
 }
+
+variable "api_url" {
+  type        = string
+  description = "URL to serve the OSV API on. Domain ownership and DNS settings has to be set up manually."
```

Comment:
What kind of manual set up is required? Can we have some documentation somewhere to explain this?

Reply:
It required verifying ownership of the test.osv.dev domain at https://www.google.com/webmasters/verification/home
and adding a CNAME entry for api.test.osv.dev pointing to ghs.googlehosted.com.

I'll note this in the terraform README